config/default.schema.json

```diff
@@ -1200,6 +1200,11 @@
 			"default": true,
 			"description": "Enables old LTI-tool tab in a course"
 		},
+		"FEATURE_SHOW_OUTDATED_USERS":{
+			"type": "boolean",
+			"default": false,
+			"description": "Shows oudated users"
```

Comment:
Add to the description please when a user is outdated. So something like... after a migration all users will be marked as outdated and this feature flag controls if they should be show or not